sensu/plugins/check-l3-router.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+HOSTNAME=`hostname`
```

Comment:
Think you may have trouble with this.  Might want to ensure that all hostnames are the same as what neutron thinks they are.